bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/details/LineItemDetailsInterface.kt

```diff
@@ -7,14 +7,14 @@ package net.gini.android.bank.sdk.capture.digitalinvoice.details
  */
 
 /**
- * Public API of the [LineItemDetailsFragment].
+ * Public API of the [DigitalInvoiceBottomSheet].
  *
  */
-interface LineItemDetailsFragmentInterface {
+interface LineItemDetailsInterface {
```

Comment:
[Request] We can remove these interfaces from the SDK's public API. We don't want clients to be able to implement these anymore. You can make them `internal`. If that's not possible, then just add `Internal use only.`. Thank you!